acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/Web3SignerRunner.java

```diff
@@ -259,4 +268,46 @@ public Path createJksCertFile(final TlsCertificateDefinition caTrustStore) {
   protected SignerConfiguration getSignerConfig() {
     return signerConfig;
   }
+
+  // Assumes database already exists, just enforces the tables exist as necessary.
+  @SuppressWarnings("UnusedVariable")
```

Comment:
Assuming the database will be created by CircleCI? Alternatively you could use embedded postgresqlfrom com.opentable.components:otj-pg-embedded that is used in the unit tests.

Reply:
Moved to embedded.